tests/conftest.py

```diff
@@ -94,116 +211,6 @@ def ftp_server_wrapper(ftp_server):
     return ftp_server
```

Comment:
I know this is an inherited issue but there are a lot of single letter variable names in `conftest.py`, could those be made more explicit?